master/internal/resourcemanagers/resource_pool.go

```diff
@@ -262,7 +262,7 @@ func (rp *ResourcePool) receiveAgentMsg(ctx *actor.Context) error {
 	switch msg := ctx.Message().(type) {
 	case sproto.AddAgent:
 		ctx.Log().Infof("adding agent: %s", msg.Agent.Address().Local())
-		rp.agents[msg.Agent] = newAgentState(msg)
+		rp.agents[msg.Agent] = newAgentState(msg, rp.config.PoolName)
```

Comment:
blocking: we shouldn't include this field because it adds unnecessary information and all `agentState` should have the same resource pool.

Reply:
Fixed